build.gradle

```diff
@@ -0,0 +1,107 @@
+allprojects  {
```

Comment:
Be sure that the code formater also runs.

Reply:
done.